dashboard/test/models/pd/attendance_test.rb

```diff
@@ -58,4 +58,13 @@ class Pd::AttendanceTest < ActiveSupport::TestCase
     assert_equal 2, teachers.count
     assert_equal [@teacher1, @unrelated_teacher], teachers
   end
+
+  test 'unique constraint' do
```

Comment:
Consider making the test name more verbose, e.g., `unique constraint on pd_session_id and teacher_id`.